jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -426,8 +427,8 @@ public void render(Context context, ProcessInstanceSummary value, SafeHtmlBuilde
                 ProcessRuntimeCSS css = ProcessRuntimeResources.INSTANCE.css();
                 String badgeHtml = null;
                 if(errCount > 0){
-                    //TODO: set link URL to correct error view perspective path
-                    String linkToErrorView = "#Error View?processInstanceId=" + value.getProcessInstanceId();
+                    //TODO: link to specific process instance ID
+                    String linkToErrorView = "#" + PerspectiveIds.EXECUTION_ERRORS;
```

Comment:
Ok, I'll try to rewrite the markup code as an Errai template.
Regarding the navigation to the error view, the final implementation depends on the new navigation framework (see [BPMSPL-635](https://issues.jboss.org/browse/) and [BPMSPL-639](https://issues.jboss.org/browse/BPMSPL-639) for example), with this PR I just wanted to provide a basic navigation to the error view (now it's merged) and avoid the application to show an error ("Error View" perspective doesn't exist)